soroban-env-host/build.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    let opt_level = std::env::var("OPT_LEVEL").unwrap_or_else(|_| "0".to_string());
+    if opt_level != "0" {
+        println!("cargo:rustc-cfg=feature=\"opt_build\"");
```

Comment:
We don't need this to be a feature. Adding it as a feature isn't really using features correctly by cargo's definition, because features are supposed to be unifiable. Not that we always use features in that way, but we actually don't need a feature in this case.

You can just just make it a custom cfg.

See https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-cfg

For example:
```suggestion
        println!("cargo:rustc-cfg=opt_build");
```

Then in the code you can write:

```rust
#[cfg(opt_build)]
```

Reply:
Cool, I kind of forgot about this possibility. The only downside is that it's not possible to actually force including this feature for debugging, but I guess temporarily removing the condition would work as well.